code/game/objects/items/cards_ids.dm

```diff
@@ -164,6 +164,16 @@
 			return
 		registered_account.payday(1)
 
+	if(href_list[VV_ID_GIVE_MINING_POINT])
+		if(!registered_account)
+			to_chat(usr, "There's no account registered!")
+			return
+		var/target_value = input(usr, "How much mining points do you want to give as? (negative value works)", "Give mining points", 100000) as num
```

Comment:
`How many mining points would you like to give? (use negative to take)`

also remove the default value